libcodechecker/libclient/product_helper.py

```diff
@@ -30,8 +31,7 @@ def __init__(self, protocol, host, port, uri, session_token=None):
         self.client = codeCheckerProductService.Client(self.protocol)
 
         if session_token:
-            headers = {'Cookie': session_manager.SESSION_COOKIE_NAME +
-                       "=" + session_token}
+            headers = {'Cookie': SESSION_COOKIE_NAME + '=' + session_token}
```

Comment:
You use this line of code in multiple places. Can we create a helper function for it?

Reply:
I don't think there exists a good place where this helper function could be put.